README.md

```diff
@@ -259,5 +260,7 @@ Questions can be asked by raising an `Issue`.
 - [Victory Chiamaka Wekwa](https://github.com/VictoryWekwa)
 - [Vinicius Ramos](https://github.com/redrookie)
 - [Vivek Singh](https://github.com/vivmost)
+- [Yashasvi Misra](https://github.com/yashasvimisra2798)
```

Comment:
I'm not sure why there was two people added. Is this a mistake? Do you have two accounts?

Reply:
Doesn't seem so cuz that person is already present at the end of the file.